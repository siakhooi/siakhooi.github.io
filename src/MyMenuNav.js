import React from 'react'
import PropTypes from 'prop-types'

MyMenuNav.propTypes = {
  displayNav: PropTypes.string,
  children: PropTypes.array
}

function MyMenuNav (props) {
  return (
    <div className="MyMenuNav" displayNav={props.displayNav}>{props.children}</div>
  )
}

export default MyMenuNav
