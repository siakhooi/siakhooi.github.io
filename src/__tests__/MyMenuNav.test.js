import React from 'react'
import renderer from 'react-test-renderer'
import MyMenuNav from '../MyMenuNav'

it('MyMenuNav/Y', () => {
  const component =
    renderer.create(
      <MyMenuNav displayNav='Y'><div>ABC</div><div>ABC</div></MyMenuNav>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

it('MyMenuNav/N', () => {
  const component =
    renderer.create(
      <MyMenuNav displayNav='N'><div>ABC</div><div>ABC</div></MyMenuNav>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
