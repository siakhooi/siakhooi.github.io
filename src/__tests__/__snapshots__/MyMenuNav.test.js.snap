// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyMenuNav/N 1`] = `
<div
  className="MyMenuNav"
  displayNav="N"
>
  <div>
    ABC
  </div>
  <div>
    ABC
  </div>
</div>
`;

exports[`MyMenuNav/Y 1`] = `
<div
  className="MyMenuNav"
  displayNav="Y"
>
  <div>
    ABC
  </div>
  <div>
    ABC
  </div>
</div>
`;
